### ISSUE TITLE
Narrator scan mode *does not work* in 1.11+

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#11488]
### Windows Terminal version (or Windows build number)

1.11+

### Steps to reproduce

Enable scan mode, or any mode, and try to navigate the buffer using narrator.

### Actual Behavior

It reads one or two things, and then gets "stuck" or moves into the titlebar.
